assets/section-password.css

```diff
@@ -324,3 +324,29 @@ details.modal .modal__toggle-open {
   vertical-align: top;
   color: rgb(var(--color-foreground));
 }
+
+.background {
+  height: 100%;
+}
+
+.background-mobile {
```

Comment:
Id maybe go with less generic class name 🤔 At least one that reflects that it's on the password page.

Reply:
Do you have any suggestions? 😅 What about `password-banner-emailsignup-background`. I feel like it's a bit too long